ndc-reference/bin/reference/main.rs

```diff
@@ -1014,17 +982,17 @@ fn eval_path_element(
         )?;
 
         for tgt_row in target.iter() {
-            if eval_column_mapping(relationship, src_row, tgt_row)? {
-                if eval_expression(
+            if eval_column_mapping(relationship, src_row, tgt_row)?
+                && eval_expression(
                     collection_relationships,
                     variables,
                     state,
-                    &predicate,
+                    predicate,
                     tgt_row,
                     tgt_row,
-                )? {
-                    matching_rows.push(tgt_row.clone());
-                }
+                )?
+            {
```

Comment:
Is this just formatting? Does clippy differ from whatever rust-analyzer does to format (I'm assuming `cargo fmt`)?

Reply:
The change is from a nested `if` to using `&&`, which means one fewer nested brace pair.